petab/v2/petab1to2.py

```diff
@@ -152,3 +183,49 @@ def _copy_file(src: Path | str, dest: Path):
             return
     except FileNotFoundError:
         shutil.copy(str(src), str(dest))
+
+
+def _melt_condition_df(
```

Comment:
Since there will be these methods for all tables, maybe a more systematic name like `v1v2_condition_df` for forward and `v2v1_condition_df` for backward conversion?